tests/channel_test.c

```diff
@@ -820,8 +820,8 @@ static bool s_shutdown_complete_pred(void *user_data) {
 static int s_test_channel_connect_some_hosts_timeout(struct aws_allocator *allocator, void *ctx) {
     (void)ctx;
 
-    aws_load_error_strings();
-    aws_io_load_error_strings();
+    aws_common_library_init();
+    aws_io_library_init();
```

Comment:
I would have aws_io_library_init() automatically call aws_common_library_init() because:
1) Saves boilerplate. You can't use a library without its dependencies anyway.
2) Lets us add new low-lever libraries later without every downstream app needing to copy/paste 1 more library to be initialized